styles/blocks/company-life/life.css

```diff
@@ -0,0 +1,26 @@
+.life {
+    display: grid;
+
+    padding: 54px 2.5%;
+
+    background-color: #ffffff;
+
+    grid-column-gap: 5%;
+    grid-row-gap: 40px;
+    justify-content: space-between;
+}
+
+.life__grid-1r {
+    grid-column: 1 / 3;
+    grid-row: 1;
+}
+
+.life__grid-2r-1c {
+    grid-column: 1;
```

Comment:
Specifying this is redundant since it follows the natural flow of grid elements.